backend/coins/btc/maketx/maketx.go

```diff
@@ -166,6 +166,7 @@ func NewTx(
 
 		inputs := make([]*wire.TxIn, len(selectedOutPoints))
 		for i, outPoint := range selectedOutPoints {
+			outPoint := outPoint
```

Comment:
fyi: this works, just because wire.NewTxIn dereferences the outpoint, making a copy.